### PR TITLE
Fix rounding error in build tests

### PR DESCRIPTION
**Pull request by justinmc**
Fix a [build error](https://build.chromium.org/p/client.flutter/builders/Mac/builds/7623/steps/steps/logs/stdio) that happened due to a rounding error in a test I just added in https://github.com/flutter/flutter/pull/26309.

### CODE REVIEW

**Comment by HansMuller** [comment]
LGTM

Since this is intended to fix the build breakage, you can land it without waiting for the complete test run.

File: packages/flutter/test/material/text_field_test.dart [hunk 1]
```diff
@@ -1757,7 +1757,7 @@ void main() {
     // line text field).
     final double lineHeight = findRenderEditable(tester).preferredLineHeight;
     scrollableState = tester.firstState(find.byType(Scrollable));
-    expect(scrollableState.position.pixels, equals(lineHeight));
+    expect(scrollableState.position.pixels, closeTo(lineHeight, 0.1));
```
Comment (Hixie) [inline packages/flutter/test/material/text_field_test.dart]: you can also use `moreOrLessEquals()`.

**Comment by Hixie** [comment]
This shouldn't be flaky, how come it passed on the PR but failed later?

**Comment by justinmc** [comment]
@Hixie Sorry I missed this comment.  It is concerning that this originally passed.  Are there any differences in how the tests are run that could produce a slightly different rounding error?  Or maybe there was code on master that I hadn't merged in that affected it slightly.  Either way it should no longer be flaky with this fix.

**Comment by Hixie** [comment]
There shouldn't be any difference. I would recommend trying to track down why there was; usually it indicates an underlying bug.

**Comment by justinmc** [comment]
On it.

**Comment by justinmc** [comment]
I couldn't find anything indicating a bug here, but I'll keep an eye out for any similar failures that might crop up.  I ran the original test a bunch of times locally and again in the build process and couldn't get it to flake.

**Pull request merged**